static/main.js

```diff
@@ -138,7 +139,7 @@ export function resetDemo() {
     const powerup_num = Math.floor(Math.random() * additional_powerups) + 1;
 
     for (let p = 0; p < powerup_num; p++) {
-      /** @type {Vector} */
+      /** @type {number} */
```

Comment:
cool, i also noticed this incorrect type and fixed this in my branch i think

Reply:
nice